aiida_wannier90/parsers.py

```diff
@@ -208,14 +208,27 @@ def raw_wout_parser(wann_out_file):
                 wf_out_i['wannier_function'] = int(
                     line.split('(')[0].split()[-1]
                 )
-                wf_out_i['spread'] = float(line.split('(')[1].split()[-1])
+                wf_out_i['spread'] = float(line.split(')')[1].strip())
                 #wf_out_i['spread'] = float(line.split()[-1])
-                #x = float(line.split()[-5].strip(','))
-                #y = float(line.split()[-4].strip(','))
-                #z = float(line.split()[-3].strip(','))
-                x = float(line.split('(')[1].split()[0].strip(','))
-                y = float(line.split('(')[1].split()[1].strip(','))
-                z = float(line.split('(')[1].split()[2].strip(','))
+                try:
+                    x = float(
+                        line.split('(')[1].split(')')[0].split(',')[0].strip()
```

Comment:
Is the final ``.strip()`` actually necessary? I think the ``float`` constructor is happy to accept string with leading or trailing whitespace.

Reply:
You are probably right, I wanted to be sure :-)